driver-core/src/test/unit/com/mongodb/internal/operation/OperationUnitSpecification.groovy

```diff
@@ -68,7 +68,7 @@ class OperationUnitSpecification extends Specification {
             [9, 0]: 25,
     ]
 
-    static int getMaxWireVersionForServerVersion(List<Integer> serverVersion) {
+    static Integer getMaxWireVersionForServerVersion(List<Integer> serverVersion) {
```

Comment:
I don't understand the warning we had for this method, but the change removed that warning. I don't want to spend more time on investigating the weirdness of Groovy.